src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
```

Comment:
It is recommended to avoid using default access modifiers. Consider defining an explicit access modifier for better code encapsulation.